### PR TITLE
e2e: use hermes master docker image

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

As requested by Romain to test the fix for https://github.com/informalsystems/hermes/issues/3580 works fine.

closes: #XXXX

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [x]  Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by damiannolan — APPROVED** [review]
Awesome, thank you @crodriguezvega and @romac for the new image! 
Do we want to merge this as is or keep it open until a release version is available?

**Comment by crodriguezvega** [comment]
> Do we want to merge this as is or keep it open until a release version is available?

I am fine merging it as is and update when v1.7 is available.

**Comment by colin-axner** [comment]
Only concern with using master is having e2e's fail suddenly, benefit of using a tag is that you don't have to worry about debugging recent changes

File: e2e/sample.config.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ chains:
 
 relayer:
   type: hermes # override with RELAYER_TYPE
-  image: colinaxner/hermes # override with RELAYER_IMAGE
-  tag: "v1.6.0" # override with RELAYER_TAG
+  image: ghcr.io/informalsystems/hermes # override with RELAYER_IMAGE
+  tag: "master" # override with RELAYER_TAG
```
Comment (chatton) [inline e2e/sample.config.yaml:26]: agree with @colin-axner , is there a non-master tag we can use that has the changes we want?

Comment (crodriguezvega) [reply e2e/sample.config.yaml:26]: There is no tag yet with the changes we need (they will be released in v1.7). I will close the PR then and wait until we have the tag. At least we tested that the fix works.

Comment (romac) [reply e2e/sample.config.yaml:26]: I just pushed that image again to Docker Hub with the [`bef2f53` tag](https://hub.docker.com/layers/informalsystems/hermes/bef2f53/images/sha256-fb4810b31be2cab16aa6cfa415b8f62aa6f82dee75bc6a14d81f38ee4f1fa1cc?context=explore).

It's the very same one as the `master` tag you were using in this PR, but that one won't change under your feet.

Comment (romac) [reply e2e/sample.config.yaml:26]: Ah sorry it's only an `arm64` image, will see if I can push the `amd64` one as well.

Comment (romac) [reply e2e/sample.config.yaml:26]: All good, it's now a multi-platform image for `linux/arm64` and `linux/amd64`, the latter image is this one https://hub.docker.com/layers/informalsystems/hermes/bef2f53/images/sha256-609720ddcd9f20fac5728187e507aa6218ddd5160ffb43cb16f65c0c9e908d79

Comment (romac) [reply e2e/sample.config.yaml:26]: Also available on GHCR.io now: https://github.com/informalsystems/hermes/pkgs/container/hermes/127899621?tag=bef2f53

Comment (crodriguezvega) [reply e2e/sample.config.yaml]: Thank you, @romac! Tag is updated now.

**Review by damiannolan — APPROVED** [review]
reACK

**Review by colin-axner — APPROVED** [review]

**Pull request merged**